### ISSUE TITLE
Alternatives for audio and video must be in the accessibility tree

### ISSUE THREAD

**Issue by kengdoj** [atbcb/ICTTestingBaseline#374]
Add checks that transcript is in the accessibility tree/available to AT.

Update [16.1 Test Procedure for Audio-only (Prerecorded)](https://ictbaseline.access-board.gov/16AudioVideo/#161-test-procedure-for-audio-only-prerecorded) and [16.2 Test Procedure for SC 1.2.1 Video-only (Prerecorded)](https://ictbaseline.access-board.gov/16AudioVideo/#162-test-procedure-for-sc-121-video-only-prerecorded) and possibly other places.

This [ACT test case](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/2eb176/d58c6252f96771666f71a65d199316108e709edd.html) fails because the transcript text is not in the accessibility tree. 

*For TT: ANDI does not detect the paragraph as hidden content since it is aria-hidden=true (not hidden with CSS).*

**Comment by kengdoj** [comment]
Indicate/explain that all things that are "check for" must be in the accessibility tree. Perhaps in introduction and include Accessibility Supported [Conformance Requirement 4](https://www.w3.org/TR/WCAG20/#conformance-reqs):

4. Only Accessibility-Supported Ways of Using Technologies: Only [accessibility-supported](https://www.w3.org/TR/WCAG20/#accessibility-supporteddef) ways of using [technologies](https://www.w3.org/TR/WCAG20/#technologydef) are [relied upon](https://www.w3.org/TR/WCAG20/#reliedupondef) to satisfy the success criteria. Any information or functionality that is provided in a way that is not accessibility supported is also available in a way that is accessibility supported. (See [Understanding accessibility support](http://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html#uc-accessibility-support-head).)

**Comment by kengdoj** [comment]
Whether visible content (like captions or transcript) are in accessibility tree are tested under 1.3.1, not in tests for captions or transcripts.
